bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/targetComponent/details2/NonSiderealNameEditor.scala

```diff
@@ -58,12 +58,18 @@ final class NonSiderealNameEditor extends TelescopePosEditor with ReentrancyHack
 
     def updateDesignation(hd: HorizonsDesignation, name: String): HS2[Unit] =
       HS2.delay {
-        val t0 = Target.name.set(spt.getTarget, name)
-        Target.horizonsDesignation.set(t0, Some(hd)).foreach(spt.setTarget)
+        Swing.onEDT {
```

Comment:
It's because of stuff everywhere listening for the data object to be updated. This manifests as calls to that go all over the place in the OT.